src/index.js

```diff
@@ -4,15 +4,19 @@ import "./index.css";
 import App from "./App";
 import { makeServer } from "./server";
 import { BrowserRouter as Router } from "react-router-dom";
+import { APIProvider } from "./context/api-context";
 
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
-    <Router>
-      <App />
-    </Router>
+    <APIProvider>
```

Comment:
Use apiprovider inside the router

Reply:
But what if Router would need any params from APIProvider ? I'm not sure rn how that would work.